src/commands/dev/lib/dev-builder.ts

```diff
@@ -152,7 +152,7 @@ export async function executeBuild(
     })
   );
 
-  match.buildResults.set(requestPath, result);
+  match.buildResults.set(requestPath || '', result);
```

Comment:
I think we shouldn't default to `''` here. Just update the `BuildResult` in types.ts to support `null` as a key for the map.

Reply:
Good call. Fixed!